packages/server/src/tests/utilities/api/query.ts

```diff
@@ -0,0 +1,36 @@
+import TestConfiguration from "../TestConfiguration"
+import { Query } from "@budibase/types"
+import { TestAPI } from "./base"
+
+export class QueryAPI extends TestAPI {
+  constructor(config: TestConfiguration) {
+    super(config)
+  }
+
+  create = async (body: Query): Promise<Query> => {
+    const res = await this.request
+      .post(`/api/queries`)
+      .set(this.config.defaultHeaders())
+      .send(body)
+      .expect("Content-Type", /json/)
+
+    if (res.status !== 200) {
```

Comment:
Why this instead of using the expected status field as we do in the other utils?

Reply:
The error message is nicer. If you use `.expect(200)` the error message doesn't show you the body of the failed request. It's been in the back of my mind for a while to try and improve this generally but I've not carved out the time yet. In a completely ideal world I'd like to even surface the error from inside the request, not catching it in any middleware, so it can be displayed in the test failure message.